RevenueCatUI/Helpers/TrialOrIntroEligibilityChecker.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  TrialOrIntroEligibilityChecker.swift
+//  
+//
+//  Created by Nacho Soto on 7/13/23.
+//
+
+import Foundation
+import RevenueCat
+
+@available(iOS 13.0, tvOS 13.0, macOS 10.15, watchOS 6.2, *)
+final class TrialOrIntroEligibilityChecker: ObservableObject {
```

Comment:
when we're done with this project, we might wanna take a look at all the SwiftUI goodies we've built for our own use and consider making them a part of the main SDK for others to benefit